client/ipa/dht.py

```diff
@@ -15,7 +15,9 @@ def __init__(self):
     self._log.info('sensor=%d pin=%d retries=%d' % (Dht._SENSOR, Dht._PIN, Dht._RETRIES))
 
   def get_sample(self):
-    humidity, temperature = Adafruit_DHT.read_retry(Dht._SENSOR, Dht._PIN, retries=Dht._RETRIES)
-    if humidity == None or temperature == None:
-      self._log.warning('failed to read temperature/humidity')
+    humidity = 3300
+    while humidity > 100:

```

Comment:
Ok... I'd rather your solution. I will give it a try whenever I have the time!

Thanks!